lib/tasks/mastodon.rake

```diff
@@ -74,6 +74,13 @@ namespace :mastodon do
       end
     end
   end
+  
+  namespace :users do
+    desc 'clear unconfirmed users'
+    task clear: :environment do
+      User.where('confirmed_at is NULL AND confirmation_sent_at <= ?', 7.days.ago).find_each(&:destroy)
```

Comment:
Hmm maybe 24 hours or 2 days? Highly unlikely that someone would press "confirm" in an e-mail after waiting for so long.